src/KOKKOS/mliap_data_kokkos.h

```diff
@@ -28,23 +28,25 @@
 namespace LAMMPS_NS {
 // clang-format off
 enum {
-  IATOMS_MASK      = 0x00000001,
-  IELEMS_MASK      = 0x00000002,
-  JATOMS_MASK      = 0x00000004,
-  JELEMS_MASK      = 0x00000008,
-  IJ_MASK          = 0x00000010,
-  BETAS_MASK       = 0x00000020,
-  DESCRIPTORS_MASK = 0x00000040,
-  EATOMS_MASK      = 0x00000080,
-  RIJ_MASK         = 0x00000100,
-  GRADFORCE_MASK   = 0x00000200,
-  GRADDESC_MASK    = 0x00000400,
-  NUMNEIGHS_MASK   = 0x00000800,
-  GAMMA_MASK_MASK  = 0x00001000,
-  GAMMA_ROW_MASK   = 0x00002000,
-  GAMMA_COL_MASK   = 0x00004000,
-  PAIR_I_MASK      = 0x00008000,
-  ELEMS_MASK       = 0x00010000,
+  IATOMS_MASK        = 0x00000001,
+  IELEMS_MASK        = 0x00000002,
+  JATOMS_MASK        = 0x00000004,
+  JELEMS_MASK        = 0x00000008,
+  IJ_MASK            = 0x00000010,
+  BETAS_MASK         = 0x00000020,
+  DESCRIPTORS_MASK   = 0x00000040,
+  EATOMS_MASK        = 0x00000080,
+  RIJ_MASK           = 0x00000100,
+  GRADFORCE_MASK     = 0x00000200,
+  GRADDESC_MASK      = 0x00000400,
+  NUMNEIGHS_MASK     = 0x00000800,
+  GAMMA_MASK_MASK    = 0x00001000,
+  GAMMA_ROW_MASK     = 0x00002000,
+  GAMMA_COL_MASK     = 0x00004000,
+  PAIR_I_MASK        = 0x00008000,
+  ELEMS_MASK         = 0x00010000,
+  MAPPED_JATOMS_MASK = 0x00010002,
```

Comment:
Why is this explicitly defined as `MAPPED_JATOMS_MASK = 0x00010002,`? I would have either expected `MAPPED_JATOMS_MASK = 0x00020000,`? or `MAPPED_JATOMS_MASK = IELEMS_MASK|ELEMS_MASK,`.

Reply:
@akohlmey I have updated the code to now be `MAPPED_JATOMS_MASK = 0x00020000`  I don't know what I was thinking when I originally wrote `MAPPED_JATOMS_MASK = 0x00010002`